kovri.md

```diff
@@ -1,64 +1,68 @@
 
-Unfortunately, Alice takes a risk to broadcast her transaction.
-Some of the nodes on the network may be logging the IP addresses that they get transaction requests from.
+Unfortunately, Alice takes a risk when broadcasting her transaction.
+Some of the nodes on the network may be logging the IP addresses they get when transaction requests are given.
 An IP address is like your home address for your internet connection.
 Attackers will try to associate transactions with the IP addresses they originate from.
 They do not know anything about what address a transaction is coming from, where it is going to, or what the amount is.
 
-However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information. 
+However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information.
 Alternatively, they could purposefully attempt to block her transaction from being broadcast to the rest of the network.
 
 So what can Alice do to mitigate these threats?
 
-Alice does not know which nodes log her IP address, so she can't simply avoid the malicious nodes.
-Her only choice is to hide her IP address.
-There are several ways to hide an IP address, including with Tor, a trusted VPN, or I2P.
-We will focus on I2P in today's video, since this is the network on which Kovri operates.
+Alice does not know which Bob logs her IP address, so she can't simply avoid the malicious Bobs.
+Her only choice is to hide her IP address from Bob.
+
+There are several ways for her to hide her IP address from Bob.
+She could use Tor or a trusted VPN; but these both have their vulnerable weaknesses.
+
+The Tor network has trusted Directory Authorities which give a handful of Tor node operators overreaching influence of network consensus.
+
+Trusted VPNs are also susceptible to trivial bandwidth correlation; making it easy for 3-letter agencies to de-anonymize her traffic.
+VPNs also adhere to local laws in order to stay in business; so Alice always runs the risk of having her user data given to a government whose policies she doesn't agree with.
+Trusted VPNs also don't protect Bob from Alice knowing who Bob is.
+
+It's all so complicated! Fortunately, Alice and Bob have a better option for protecting their privacy. Their option is Kovri.
 
-In a sentence, Kovri is a lightweight router compatible with cryptocurrencies and built for the I2P network.
-I2P stands for Invisible Internet Protocol.
-I2P is a layer of the internet that allows for all connections in the network to be private.
-In contrast with regular internet routing or TOR's onion routing, I2P utilizes what it refers to as garlic routing.
-Instead of connecting with participants directly or through TOR nodes, such as going directly to a website, all information is encrypted and relayed through other I2P routers in the network.
+Kovri is a free, decentralized, anonymity technology developed by Monero.
```

Comment:
"Kovri is a free, decentralized anonymity technology developed by Monero."

Remove the comma

Reply:
Actually, this is perfectly valid English and does not reflect on the spoken language. If the reader is pedantic, he may interpret otherwise. And that's good if he does, we need the space for breath.